components/fxa-client/src/oauth/attached_clients.rs

```diff
@@ -9,6 +9,7 @@ use crate::{error::*, util, CachedResponse, FirefoxAccount};
 const ATTACHED_CLIENTS_FRESHNESS_THRESHOLD: u64 = 60_000; // 1 minute
 
 impl FirefoxAccount {
+    /// Fetches the list of attached clients connected to the current account.
```

Comment:
FYI: This is an unrelated addition. Just adding it because I was in the neighborhood.